app/models/message_draft.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+# Message drafts are essentially messages with an implied folder (drafts)
+class MessageDraft < Message
+end
```

Comment:
if you include ActiveModel::Validations, you can validate the attributes like they are ActiveRecord objects. Then in your controller, only try to send to MHV if .valid? otherwise raise the common exception for model validations.